AUDIO/GetSoundId.md

```diff
@@ -8,5 +8,8 @@ ns: AUDIO
 int GET_SOUND_ID();
 ```
 
+```
+Returns numbers in order until it reaches 100, when it reaches 100 it starts returning -1. This native appearantly blocks sounds played with PLAY_SOUND_FRONTEND; PLAY_SOUND; PLAY_SOUND_FROM_COORD; PLAY_SOUND_FROM_ENTITY.
```

Comment:
I'm confused, what is actually meant here by 'blocks sounds'? It looks like this returns a usable sound id in order to play a sound. Greater than `100` returns `-1` as you are saying, that's correct 🙂

It can be used in conjunction with `PLAY_SOUND` and other related audio natives that take a sound id parameter.